### PR TITLE
cmd/storagenode: pull more things into releaseDefaults

### PR DESCRIPTION
**Pull request by jtolds**
What: Move more of the release defaults out of docker-specific code into the main release defaults system

Why: So things work better without docker

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: storagenode/piecestore/endpoint.go [hunk 1]
```diff
@@ -42,11 +42,16 @@ var _ pb.PiecestoreServer = (*Endpoint)(nil)
 type OldConfig struct {
 	Path string `help:"path to store data in" default:"$CONFDIR/storage"`
 
-	WhitelistedSatelliteIDs string        `help:"a comma-separated list of approved satellite node ids" default:""`
-	SatelliteIDRestriction  bool          `help:"if true, only allow data from approved satellites" devDefault:"false" releaseDefault:"true"`
-	AllocatedDiskSpace      memory.Size   `user:"true" help:"total allocated disk space in bytes" default:"1TB"`
-	AllocatedBandwidth      memory.Size   `user:"true" help:"total allocated bandwidth in bytes" default:"500GiB"`
-	KBucketRefreshInterval  time.Duration `help:"how frequently Kademlia bucket should be refreshed with node stats" default:"1h0m0s"`
+	WhitelistedSatelliteIDs string `help:"a comma-separated list of approved satellite node ids" devDefault:"" releaseDefault:"
```
Comment (stefanbenten) [inline storagenode/piecestore/endpoint.go]: Lets remove your SatelliteID @jtolds

Comment (mniewrzal) [reply storagenode/piecestore/endpoint.go]: @stefanbenten can you remove JT satellite from the list? then we can merge it.

Comment (stefanbenten) [reply storagenode/piecestore/endpoint.go]: Yes, let me do that now

Comment (jtolds) [reply storagenode/piecestore/endpoint.go]: sorry i'm useless!

**Pull request merged**